Fix pyre error for logits

iseeyuan opened the pull request:
Summary: Fix the pyre error when logits is not initialized

Reviewed By: swolchok

Differential Revision: D70183946

jackzhxng reviewed (approved):
Trusting @swolchok's internal review

Commit:
Fix pyre error for logits (#8687)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/8687

Fix the pyre error when logits is not initialized

Reviewed By: swolchok, jackzhxng

Differential Revision: D70183946
